torchbenchmark/util/experiment/metrics.py

```diff
@@ -38,19 +40,58 @@ def get_latencies(func, device: str, nwarmup=WARMUP_ROUNDS, num_iter=BENCHMARK_I
         result_summary.append((t1 - t0) / NANOSECONDS_PER_MILLISECONDS)
     return result_summary
 
+def get_peak_memory(func, device: str, num_iter=BENCHMARK_ITERS, export_metrics_file='', metrics_needed=[], metrics_gpu_backend='dcgm') -> Tuple[float, float]:
```

Comment:
I guess we don't need to run 15 iterations to get the memory metrics?
Since we plan to print this metrics by default, it is better to save some time.

Reply:
I use a new variable `MEMPROF_ITER` and set it to 2. It's better to set a bigger number like 4 if we want to get more stable and accurate results since some models' execution times are too short.